.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [ "*" ]
+    branches: [ "**" ]
```

Comment:
seems like this change is on the other branch as well, the difference is that this one is lacking context.

Reply:
So, I changed this so it can run the jobs on this branch. I know I put it in other branches, but before I added this when making initial commits, it was not performing the CI tests. I had to look up glob patterns and found that any branch with the slash (/) would be ignored. So I changed it here to ensure the CI would pass tests on this branch.